lib/resources/platform.rb

```diff
@@ -46,8 +50,19 @@ def in_family?(family)
       @platform.family_hierarchy.include?(family)
     end
 
-    def families
-      @platform.family_hierarchy
+    def params
+      h = {
+        name: name,
+        families: families,
+      }
+
+      # Avoid adding Release/Arch for APIs
+      unless in_family?('api')
+        h[:release] = release
```

Comment:
According to https://github.com/chef/inspec/issues/1661 section 5. The platform resource release is not optional. Do we have a sdk version that we could use for release of an api?

Reply:
I can throw in a train PR to add the sdk version as the release for the transports then it will be available here.

Reply:
Cool beans, thanks @jquick. I'll move release out of the omit category.

Reply:
Fields added via https://github.com/chef/train/pull/258